packages/terra-switch/src/Switch.module.scss

```diff
@@ -0,0 +1,199 @@
+// Themes
+@import './clinical-lowlight-theme/Switch.module';
+@import './orion-fusion-theme/Switch.module';
+
+:local {
+  .switch {
+    background-color: transparent;
+    border: 0;
+    display: flex;
+    flex-direction: row;
+
+    &:hover {
+      background-color: transparent;
+      border: 0;
+
+      .slider:not(.is-disabled) {
+        background-color: var(--terra-switch-slider-hover-background-color, #fff);
+        border: var(--terra-switch-slider-hover-border, 0);
+        box-shadow: var(--terra-switch-slider-hover-box-shadow, 2px 0 2px 0 rgba(0, 0, 0, 0.2), -2px 0 2px 0 rgba(0, 0, 0, 0.2), 
+        1px 0 0 0 #909496, 0 0 4px 1px rgba(0, 0, 0, 0.2));
+      }
+
+      .slider.is-selected:not(.is-disabled) {
+        background-color: var(--terra-switch-slider-selected-hover-background-color, #fff);
+        border: var(--terra-switch-slider-selected-hover-border, 0);
+        box-shadow: var(--terra-switch-slider-selected-hover-box-shadow, 2px 0 2px 0 rgba(0, 0, 0, 0.2), -2px 0 2px 0 rgba(0, 0, 0, 0.2), 
+        -1px 0 0 0 #01639e, 0 0 4px 1px rgba(0, 0, 0, 0.2));
+      }
+    }
+
+    &:active {
+      .slider:not(.is-disabled) {
+        background-color: var(--terra-switch-slider-active-background-color, #fff);
+        border: var(--terra-switch-slider-active-border, 0);
+        box-shadow: var(--terra-switch-slider-active-box-shadow, inset 0 0 0 1px #fff, inset 5px 2px 4px 2px #e6e7e8, 2px 0 2px 0 rgba(0, 0, 0, 0.2),
+        -2px 0 2px 0 rgba(0, 0, 0, 0.2), 1px 0 0 0 #909496, 0 0 4px 1px rgba(0, 0, 0, 0.2)); // pressed state.
+      }
+
+      .slider.is-selected:not(.is-disabled) {
+        background-color: var(--terra-switch-slider-selected-active-background-color, #fff);
+        border: var(--terra-switch-slider-selected-active-border, 0);
+        box-shadow: var(--terra-switch-slider-selected-active-box-shadow, inset 0 0 0 1px #fff, inset 5px 2px 4px 2px #e6e7e8, 2px 0 2px 0 rgba(0, 0, 0, 0.2),
+        -2px 0 2px 0 rgba(0, 0, 0, 0.2), -1px 0 0 0 #01639e, 0 0 4px 1px rgba(0, 0, 0, 0.2)); // pressed state.
+      }
+    }
+
+    &:focus {
+      background-color: transparent;
+      border: 0;
+      outline: 0;
+
+      &[data-terra-switch-show-focus-styles='true'] .slider:not(.is-disabled) {
```

Comment:
If focus is no longer possible with the `isDisabled` check in the javascript, we can remove this qualifiers for `not(.is-disabled)`

Reply:
this we can remove only whole component was just a button but now after changing this to div we still need to prevent focus styles for disabled switch.